### DISCUSSION TITLE
Link Examples in Docs to actual code

### DISCUSSION THREAD

**Discussion by fakoredeDamilola** [icflorescu/mantine-datatable#528] [Ideas]
I have been looking through the documentation and could not find where the components used were destructured. I had to go to github. 
You have the code in the docs...that is awesome, but the import statement is not added.

**Comment by icflorescu** [comment]
The import statements (and other irrelevant lines of code) are omitted for brevity in the documentation website examples, so you can focus on what's important.
It may be a good idea, however, to add links to the full source.
So, I'll keep that in mind and try to come up with a solution.

Thanks for the suggestion!

**Comment by icflorescu** [comment]
I couldn't find the time to address this, and since it's not top priority I'll be moving it to discussions.

I'm open if somebody wants to step in and come up with a solution and a PR to address this. I.e. maybe rewrite the code example components to render links to the actual example sources?...

I'd be reluctant to show **all** the examples code for the aforementioned reason: the import statements (and other irrelevant lines of code) are omitted for brevity in the documentation website examples, so users can **focus on what's important**. Doing otherwise would enable people to simply copy-paste without thinking and understanding, and I'm not a huge fan of that.
